data_x/config/model.py

```diff
@@ -4,12 +4,13 @@
 
 
 class Model:
-    def __init__(self, name, type, variant, task, params):
+    def __init__(self, name, type, variant, task, params, label):
         self.name = name
         self.type = type
         self.variant = variant
         self.task = task
         self.params = params
+        self.label = label
```

Comment:
instead of introducing a new config var could we use task 😄 ?

Reply:
Updated -